nixos/modules/services/networking/tailscale.nix

```diff
@@ -39,6 +39,9 @@ in {
         StateDirectory = "tailscale";
         StateDirectoryMode = 700;
 
+        CacheDirectory = "tailscale";
+        CacheDirectoryMode = 700;
```

Comment:
Upstream sets this to 0750:

https://github.com/tailscale/tailscale/commit/362d6af4e235396bd555b9175c7426063ee50404#diff-6b3b7b8c67cf3362b5c4f99a65f158ccR20

Reply:
Fixed (and also StateDirectoryMode). Doesn't matter hugely, but it's nice to be consistent with upstream.

Reply:
Thanks!